### DISCUSSION TITLE
Is there any way to reuse previously fuzzed data?

### DISCUSSION THREAD

**Discussion by tvistoman** [jtpereyda/boofuzz#573] [Q&A]
In my code
...
session.connect(first)
session.connect(first,second)
session.connect(second, third)
session.fuzz()
...

it the third test case i'm getting some fuzzed data, is there any way to store it in some variable and use later in the second test case like a 'Static' on the next fuzzing pass?

**Comment by jtpereyda** [comment]
Do you want to reuse data that the fuzzer is generating, or data that's being received from the target?

**Comment by tvistoman** [comment]
I want to use data generated by the fuzzer.

**Comment by jtpereyda** [comment]
Well you could view the log via the web UI or via dumping the log to the console, in which case the sent values are printed. I guess it prints the entire message, so you'd need to grab the relevant excerpt. They are printed as Python bytes literals, so you can just grab the string and copy it into your code. Is that the sort of thing you have in mind?
